Backend/routes/auth/auth.ts

```diff
@@ -0,0 +1,28 @@
+import { Router, Request, Response } from "express";
+import passport from "passport";
+import signup from "../../controllers/auth/signup";
+import login from "../../controllers/auth/login";
+
+export type UserType = {
+    id?: number;
+    firstName: string;
+    lastName: string;
+    email: string;
+    password: string;
+};
+
+const router = Router();
+
+router.post("/signup", signup);
+router.post("/login", login);
+
+// Demo protected route to test passport js functionality
+router.get(
+    "/secret",
+    passport.authenticate("jwt", { session: false }),
```

Comment:
what does `session:false` do?

Reply:
Password js by default tries to create a session for user after authenticating. It is recommended to set it to false if we don't create persistent server side sessions for users.